cmd/earthly/build_cmd.go

```diff
@@ -243,7 +250,7 @@ func (app *earthlyApp) actionBuildImp(cliCtx *cli.Context, flagArgs, nonFlagArgs
 		runnerName = fmt.Sprintf("local:%s", hostname)
 	} else {
 		if app.satelliteName != "" {
-			runnerName = fmt.Sprintf("sat:%s/%s", app.orgName, app.satelliteName)
+			runnerName = fmt.Sprintf("sat:%s/%s", app.cliFlags.orgName, app.satelliteName)
```

Comment:
I'm curious about the reason for explicitly referencing the embedded struct here?

Reply:
It's because I added the same field in https://github.com/earthly/earthly/pull/2658/files#diff-c5311aaeb839beacd8c0108569b5e9330f54233d2b80b16093f6e492782ca4c5R181 and it caused a collision

Reply:
ah ha! That wasn't immediately obvious to me. Moving those fields to the new `analytics.ProjectTracker` struct will prevent this collision.